src/utilities/theme/context.tsx

```diff
@@ -1,10 +1,4 @@
 import React from 'react';
 import {Theme} from './types';
 
-export interface ThemeContextType {
-  logo: Theme['logo'] | null;
-}
-
-export const ThemeContext = React.createContext<ThemeContextType>({
-  logo: null,
-});
+export const ThemeContext = React.createContext<Theme | undefined>(undefined);
```

Comment:
Curious why we're deciding to make this undefined rather than a base object?

Reply:
I'm trying to make consistent behaviours between all the contexts that get provided by the AppProvider (I18nContext, ScrollLockManagerContext, StickyManagerContext, AppBridgeContext, LinkContext, ThemeContext - via ThemeProvider).

Currently we say "you must wrap everything in an AppProvider", and there is no expectation for a component to work if you don't do so. The AppProvider provides the default values for all these contexts. It didn't seem like there was any value in providing a default within ThemeProvider, and then specifying one here as the default of the context too (which would never get used because the default from ThemeProvider would always overwrite it).

---

I'd be interested in talking through changing that assumption and identifying contexts that can and should provide their own defaults instead of being provided by the AppProvider/ThemeProvider instead of setting them within the AppProvider/ThemeProvider (and thus allowing some components to work without being wrapped in an AppProvider/ThemeProvider), but we should think about all these contexts at the same time.

Reply:
> I'd be interested in talking through changing that assumption and identifying contexts that can and should provide their own defaults instead of being provided by the AppProvider/ThemeProvider instead of setting them within the AppProvider/ThemeProvider (and thus allowing some components to work without being wrapped in an AppProvider/ThemeProvider), but we should think about all these contexts at the same time.

That's definitely something to look at in the future. For the time being, as long as we throw an error when `!theme` this should be ok. My view was that theme was a constructed object that'll have standard defaults whereas `sticky-manager` has an instance of an object that's specific to the state of the consumer's app. Since we do have the premise that we'll always have `AppProvider` within the react tree and will throw an error when we don't. I'm ok with this 👍